index.html

```diff
@@ -758,7 +757,7 @@
     <pre class="highlight">
 navigator.nfc.watch((message) => {
   for (let record of message.data) {
-    let value = JSON.parse(record.data);
+    let value = record.data;
     let article =/[aeio]/.test(value.title) ? "an" : "a";
     console.log(`$(value.name) is $(article) $(value.title)`);
```

Comment:
let article =/[aeio]/.test(record.data.title) ? "an" : "a";
     console.log(`$(record.data.name) is $(article) $(record.data.title)`);